modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase

```diff
@@ -55,6 +55,83 @@ definition {
 			name = "Channels");
 	}
 
+	@description = "This is a test for COMMERCE-10610. As a buyer user, I can view the default channels with no results using the view permissions for products and catalogs of the admin panel"
+	@priority = 3
+	test CanViewDefaultNoChannelResultsWithViewPermissionsForProductsAndCatalogs {
+		property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}company.security.strangers.verify=false";
+		property portal.acceptance = "false";
+
+		task ("Given an account with a buyer user is created on Minium") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
```

Comment:
I don't think we need to create an account with a buyer user, just creating a regular user is enough. Thanks!